release_notes/topics/4_6_0_final.adoc

```diff
@@ -9,4 +9,11 @@ Keycloak Gatekeeper provides a security proxy that can be used to secure applica
 It can be installed locally alongside your application or as a sidecar on OpenShift or Kubernetes.
 
 Huge thanks to <a href="https://github.com/gambol99">gambol99</a> for contributing this work to Keycloak.
+endif::[]
+
+= Rules/Drools Policy Marked as a Technology Preview Feature
+
+Until now, Drools policies were enabled by default. But now, this policy type is only available as a technology preview feature and
+to use it you need to enable the preview profile or the corresponding feature. Take a look at the link:{authorizationguide_link}[{authorizationguide_name}] for more details.
+
 endif::[]
```

Comment:
@pedroigor It looks like we have two `endif::[]` statements in here, your new one and an existing one on line 19. Am I reading that wrong?